artemis-server/src/main/java/org/apache/activemq/artemis/core/management/impl/QueueControlImpl.java

```diff
@@ -2048,6 +2048,37 @@ public void deliverScheduledMessages(String filter) throws Exception {
       }
    }
 
+   @Override
+   public String verifyConnections() throws Exception {
```

Comment:
hmmm... I need a better name for this method...

getInvalidConnections()
inspectInvalidConnections

any other ideas?

Reply:
I'm enhancing listConsumerAsJSON for this...

thanks to @jbertram idea